toothpick-runtime/src/main/java/toothpick/Toothpick.java

```diff
@@ -39,12 +39,31 @@
   // its transformation
   private static final ConcurrentHashMap<Object, Scope> MAP_KEY_TO_SCOPE =
       new ConcurrentHashMap<>();
+  private static final ConcurrentHashMap<Object, Scope> ROOT_SCOPES = new ConcurrentHashMap<>();
   static Injector injector = new InjectorImpl();
 
   protected Toothpick() {
     throw new RuntimeException("Constructor can't be invoked even via reflection.");
   }
 
+  /**
+   * @return The root scope as follows: <br>
+   *     <ul>
+   *       <li>If there's an existing scope tree the root scope of that tree
+   *       <li>If there's no tree it will create a default root scope and return it
+   *       <li>If there are multiple scope trees (e.g. a forest) this method will throw an exception
+   *     </ul>
```

Comment:
The javadoc does not match the style of the rest of the file. 

I'd have an explanation first, where you provide much of the details. Then the return value below with simple explanation to identify it. But the majority is in the details above.